bodhi/client/__init__.py

```diff
@@ -370,11 +371,23 @@ def edit(user, password, url, **kwargs):
             resp = client.query(**query_param)
             title = resp['updates'][0]['title']
         elif re.search(bindings.UPDATE_TITLE_RE, kwargs['update']):
+            query_param = {'like': kwargs['update']}
+            resp = client.query(**query_param)
             title = kwargs['update']
         del(kwargs['update'])
         kwargs['builds'] = title
         kwargs['edited'] = title
 
+        # Convert list of 'Bug' objects in DB to comma separated bug_ids for parsing.
+        # Creating deepcopy prevents original object to be modified (useful in test environment).
+        former_update = copy.deepcopy(resp['updates'][0])
+        former_update['bugs'] = ",".join([str(bug['bug_id']) for bug in former_update['bugs']])
```

Comment:
you might be able to avoid to do the deepcopy, but doing this in the for loop. That way you would populate ```kwargs['bugs']``` instead of ```former_update['bugs']```